dev-preview/README.md

```diff
@@ -97,6 +97,48 @@ Installation instructions can be found in the [repo](https://github.com/flightct
 
 Instructions for managing fleets can be found [here](https://github.com/flightctl/flightctl/blob/main/docs/user/managing-fleets.md) while for managing devices are available [here](https://github.com/flightctl/flightctl/blob/main/docs/user/managing-devices.md). One of the benefits of using Edge Management together with ACM is the ability to enroll devices running Red Hat Device Edge seamlessly.
 
+## Openshift Cluster API Operator
+
+The Cluster API Operator is a Kubernetes Operator designed to empower cluster administrators to handle the lifecycle of Cluster API providers within a management cluster using a declarative approach. It aims to improve user experience in deploying and managing Cluster API, making it easier to handle day-to-day tasks and automate workflows with GitOps.
+
+OpenShift (OCP) Cluster API Operator helm chart is based on the k8s-sig cluster-api-operator helm chart. The OpenShift helm chart uses the redhat-registry container images to deploy the cluster-api-operator, cluster-api and cluster-api-aws-providers. 
+
+#### Note:
+
+As constrain for the OpenShift (OCP) Cluster API Operator helm chart devPreview release; It is not supported to use the MCE (HyperShift) operator to provision a host control plan cluster while using the OCP cluster API operator helm chart.
+
+### Installation
+Add CAPI Operator & cert manager helm repository:
+```
+$ helm repo add capi-operator https://raw.githubusercontent.com/openshift/cluster-api-operator/refs/heads/main/openshift
+$ helm repo add jetstack https://charts.jetstack.io --force-update
```

Comment:
Does the flow work with the Cert-Manager operator???  This can be okay I guess for devPreview, but definitely not for TP or GA.

Reply:
ya, it should work with our downstream cert-manager https://github.com/openshift/cert-manager-operator. It is based on the upstream one https://docs.openshift.com/container-platform/4.17/security/cert_manager_operator/index.html